src/boot/efi/UEFI_SECURITY.md

```diff
@@ -4,6 +4,13 @@ PE binary, adding various features, `systemd-stub`. These components fully suppo
 this document will describe their security posture and how they comply with industry-standard expectations
 for UEFI SecureBoot workflows.
 
+Note that `systemd-stub` is not the same, or an alternative, to the Linux kernel's own EFI stub. The kernel
+stub's role is to prepare to launch the 'proper' kernel execution from EFI. `systemd-stub` on the other hand
+loads various resources, including the kernel image, via the EFI LoadImage/StartImage protocol (although it
+does support the legacy handover protocol, as a fallback for older kernels on x86). The purpose of
+`systemd-stub` is to provide additional features and functionality for both `systemd-boot` and `systemd` in
+userspace.
```

Comment:
`systemd-stub` offers nothing really for `systemd-boot`.


Maybe say: The purpose of `systemd-stub` is to provide additional features and functionality facing `systemd` in the OS userspace.

Reply:
it takes stuff passed from sd-boot via its protocol/variables, like kernel command line, so I'd say it is accurate that its purpose is to work with both components

Reply:
there's nothing sd-boot specific in there.

Reply:
Ok, Boot Loader Interface specific then? This is an explanatory paragraph added after direct feedback that things weren't clear, so I am more concerned in ensuring a non-expert reader gets the right kind of idea of the split and relationship between these components, than being strictly adherent to the letter of the definitions (in this paragraph), but if you want to improve the wording please feel free to send a follow-up